.github/workflows/push-1password-secrets-to-gha.yaml

```diff
@@ -0,0 +1,45 @@
+name: "Push secrets from 1Password to Github Actions"
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - "push-secrets-*"
+
+jobs:
+  push-secrets:
+    name: "Push Secret"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install 1Password CLI
+        uses: 1password/install-cli-action@v1
+
+      - name: Load GH token
+        id: op-gh-token
+        uses: 1password/load-secrets-action@v2
+        with:
+          export-env: false
+        env:
+          OP_SERVICE_ACCOUNT_TOKEN: ${{ secrets.OP_SERVICE_ACCOUNT_TOKEN }}
+          GITHUB_TOKEN: op://pulumi/gh-pulumi-runner-token/credential
+
+      # - uses: pulumi/auth-actions@v1
```

Comment:
Once cf has a pulumi org setup, we should setup OIDC. Then this piece can get uncommented and the `GITHUB_TOKEN` below can be removed